.settings/keycloak/Dockerfile

```diff
@@ -6,14 +6,17 @@ ARG DOCKERIZE_VERSION=v0.6.1
 
 ENV KEYCLOAK_URL=$KEYCLOAK_URL
 
+# TODO: I didn't need this for my build - this needs to be verified on a windows box?
```

Comment:
Should we start creating JIRA tasks on our board for TODOs?

Reply:
Is there a way to automate that?

Reply:
given that it's government JIRA, most likely not lol

Reply:
nope ;-( no automation